src/components/screens/vacation-requests-screen.tsx

```diff
@@ -282,15 +303,101 @@ const VacationRequestsScreen = () => {
     }
   };
 
+  /**
+   * Get updated vacation requests statuses
+   *
+   * @param newStatus vacation request status
+   * @param selectedRowIds selected row ids
+   * @returns updated vacation request statuses
+   */
+  const getUpdatedVacationRequestStatuses = async (
```

Comment:
Perhaps this name is somewhat misleading as you are actually updating or creating requestStatuses. Also as you return the updated statuses but not the newly created ones does this actually work as intended in the updateVacationRequestStatuses function. I suppose it does as the createVacationRequestStatus updates the latestVacationRequestStatuses. Perhaps its a bit of a jigsaw but maybe this is fine. Just consider the rename

Reply:
I decided to make a separate function for getting the created vacation request statuses, solving this issue.